modules/videoio/src/cap.cpp

```diff
@@ -117,6 +117,7 @@ bool VideoCapture::open(const String& filename, int apiPreference, const std::ve
 
     const VideoCaptureParameters parameters(params);
     const std::vector<VideoBackendInfo> backends = cv::videoio_registry::getAvailableBackends_CaptureByFilename();
+    const std::vector<VideoDeprecatedBackendInfo> deprecatedBackends = cv::videoio_registry::getDeprecatedBackends();
```

Comment:
Not used anymore.

Function itself is not needed.

Structure should be dropped too.